docs/sdk/cosmos-sdk-cli.md

```diff
@@ -1,19 +1,18 @@
-# cosmos-sdk-cli 
-Create a new blockchain project based on cosmos-sdk with a single command.
+# cosmos-sdk-cli
```

Comment:
these are the titles that show up in the sidebar, would be good to pretty-fy it

Reply:
I'm good with the current naming. Having a section named after a specific tool is OK right?

Reply:
`# Cosmos SDK CLI`